frontend/src/cards/DisparityBarChartCard.tsx

```diff
@@ -82,7 +82,9 @@ function DisparityBarChartCardWithKey(props: DisparityBarChartCardProps) {
     locationName
   );
 
-  const filename = `${metricConfig.populationComparisonMetric?.chartTitle}${locationName}`;
+  const filename = `${metricConfig.populationComparisonMetric?.chartTitleLines.join(
```

Comment:
it seems like we shouldn't have to duplicate effort making this joined filename and making the joined titles elsewhere.... can we just make it once when the component loads and then use that "chartTitleOneLine" string as the filename, the missing data text, and wherever else a one-liner version of the title is needed?